From bd6020f1db429e9e129ebc21875773d68b4a1e50 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Tue, 8 Oct 2013 20:06:24 +0000
Subject: [PATCH 119/462] screenshot.lib

V1.0.0.0
Mantis 0028892: PLF: SAM-Hotkey not working
convert screenshot.exe to screenshot.lib
and integrate screenshot.lib into appcontrol.exe


git-svn-id: svn://localhost/SelfServiceBuild/trunk@176 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/cpp/makefile.common.mak | 2 ++
 SelfServiceBuild/Build/cpp/makefile.mak        | 1 +
 2 files changed, 3 insertions(+)

diff --git a/SelfServiceBuild/Build/cpp/makefile.common.mak b/SelfServiceBuild/Build/cpp/makefile.common.mak
index eb8c9b6..15d1bc1 100644
--- a/SelfServiceBuild/Build/cpp/makefile.common.mak
+++ b/SelfServiceBuild/Build/cpp/makefile.common.mak
@@ -294,6 +294,7 @@ LIB_DST_MSYSTOOLS         = $(_LIB)\msystools.lib
 LIB_DST_INITIDL           = $(_LIB)\initIDL.lib
 LIB_DST_TEXTIDL           = $(_LIB)\textIDL.lib
 LIB_DST_TEXTIDLS          = $(_LIB)\textIDLs.lib
+LIB_DST_SCREENSHOT        = $(_LIB)\screenshot.lib
 
 DLL_ALERTS                = $(_DLL)\alerts.dll
 DLL_CXAPI                 = $(_DLL)\cxapi.dll
@@ -346,6 +347,7 @@ LIB_MSYSTOOLS             = $(LIB_ASSO) $(LIB_GRAPHIC) $(LIB_TEXT) $(LIB_SYSTOOL
 LIB_INITIDL               = $(LIB_DST_INITIDL)
 LIB_TEXTIDL               = $(LIB_DST_TEXTIDL)
 LIB_TEXTIDLS              = $(LIB_DST_TEXTIDLS)
+LIB_SCREENSHOT            = $(LIB_DST_SCREENSHOT) $(LIB_FREEIMAGES)
 
 LIB_TRACE                 = $(_LIB)\trace.lib
 LIB_TRACEHDL              = $(_LIB)\tracehdl.lib
diff --git a/SelfServiceBuild/Build/cpp/makefile.mak b/SelfServiceBuild/Build/cpp/makefile.mak
index 8ec3613..d6ef456 100644
--- a/SelfServiceBuild/Build/cpp/makefile.mak
+++ b/SelfServiceBuild/Build/cpp/makefile.mak
@@ -70,6 +70,7 @@ COMMON_DIRECTORIES= \
  Massai\cpp\systools \
  Massai\cpp\graphic \
  Massai\cpp\mSystools \
+ Massai\cpp\ScreenShot \
  Massai\idl\Massai \
  Services\cpp\CxComm\dll \
  Massai\cpp\MassaiTools \
-- 
2.41.0.windows.1

